test/e2e/webdriver/driver.js

```diff
@@ -259,9 +258,47 @@ class Driver {
     }, this.timeout);
   }
 
-  async waitForElementNotPresent(rawLocator) {
+  /**
+   * Wait until an element is absent.
+   *
+   * This function MUST have a guard to prevent a race condition. For example,
+   * when the previous step is to click a button that loads a new page, then of course
+   * during page load, the rawLocator element will be absent, even though it will appear
+   * a half-second later.
+   *
+   * The first choice for the guard is to use findElement before this statement on an element
+   * that will appear at the same time as the rawLocator element.
+   *
+   * The second choice for the guard is to use the waitAtLeast parameter.
+   *
+   * @param {*} rawLocator
+   * @param {number} waitAtLeast - The minimum milliseconds to wait before passing
+   * @param {number} timeout - The maximum milliseconds to wait before failing
+   */
+  async waitForElementNotPresent(
```

Comment:
Nit: After checking your implementation, i think we can even pass another parameter as the guard selector, which is optional. So we can execute `findElement` of guard and then execute the not present await. what's your thoughts?

Reply:
Yeah I considered that, do you think it would be better?  Would it use a mix of positional and named parameters like this?

```
async waitForElementNotPresent(
  rawLocator,
  { findElementGuard = null, waitAtLeastGuard = 0, timeout = this.timeout },
) {
```

Reply:
Looks clear to me to separate them like that !